### PR TITLE
Refs #30331 - move layout proptypes to helper

### PR DESCRIPTION
**Pull request by MariaAga**
Split from https://github.com/theforeman/foreman/pull/7805
To make it easier to reuse the prop types for sub components

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/Layout/components/TaxonomyDropdown.js [hunk 1]
```diff
@@ -101,12 +101,12 @@ TaxonomyDropdown.propTypes = {
...
   currentTaxonomy: PropTypes.string.isRequired,
   taxonomies: PropTypes.arrayOf(
-    PropTypes.shape({
-      id: PropTypes.number.isRequired,
-      title: PropTypes.string,
-      href: PropTypes.string.isRequired,
+    PropTypes.shape({	
+      id: PropTypes.number.isRequired,	
+      title: PropTypes.string,	
+      href: PropTypes.string.isRequired,
```
Comment (xprazak2) [inline webpack/assets/javascripts/react_app/components/Layout/components/TaxonomyDropdown.js]: nit: trailing whitespace

**Comment by xprazak2** [comment]
Looking good overall, Travis failures might be related.

**Comment by MariaAga** [comment]
Tests should work now - added fixtures

**Review by xprazak2 — APPROVED** [review]

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**